src/components/debug.rs

```diff
@@ -33,6 +40,8 @@ impl DebugInfo {
             terminal_width: 0,
             terminal_height: 0,
             gameday_active_views: GamedayPanels::default(),
+            date: NaiveDate::from_ymd_opt(2022, 07, 09).unwrap(),
+            stat_type: StatGroup::Pitching
```

Comment:
I think these should use default values: `NaiveDate::default()` and `StatGroup::default()`. You'll need to add a `Default` derive to StatGroup